### DISCUSSION TITLE
Autofocusser for all sky camera

### DISCUSSION THREAD

**Discussion by baryah** [aaronwmorris/indi-allsky#1120] [Ideas]
Can an 'autofocusser' e.g. myFocuserPro2 (https://www.indilib.org/focusers/myfocuser-pro-2.html) be connected to Indi-allsky.

**Comment by aaronwmorris** [comment]
The auto focuser could be connected to the indiserver, but indi-allsky will not manage it.  You would have to manage the auto-focuser via another mechanism.
